ipwb/indexer.py

```diff
@@ -306,19 +304,18 @@ def getCDXJLinesFromFile(warcPath, **encCompOpts):
 
             objJSON = json.dumps(obj)
 
-            cdxjLine = '{0} {1} {2}'.format(originaluri_surted,
-                                            timestamp, objJSON)
+            cdxjLine = f'{originaluri_surted} {timestamp} {objJSON}'
             cdxjLines.append(cdxjLine)  # + '\n'
         return cdxjLines
 
 
 def generateCDXJMetadata(cdxjLines=None):
     metadata = ['!context ["http://tools.ietf.org/html/rfc7089"]']
     metaVals = {
-        'generator': "InterPlanetary Wayback v.{0}".format(ipwbVersion),
-        'created_at': '{0}'.format(datetime.datetime.now().isoformat())
+        'generator': f'InterPlanetary Wayback {ipwbVersion}',
+        'created_at': f'{datetime.datetime.now().isoformat()}'
```

Comment:
I think the `isoformat()` already returns a string, so no need to perform an unnecessary interpolation.

```suggestion
        'created_at': datetime.datetime.now().isoformat()
```